src/Base/PyTools.c

```diff
@@ -660,8 +661,8 @@ PP_Debug_Bytecode(PyObject *codeobject, PyObject *moddict)
     res = PP_Run_Function(            /* "pdb.runeval(codeobj, gdict, ldict)" */
              "pdb",    "runeval",     /* accepts string|code, code=stmt|expr  */
              "O",      &presult,
-             "(OOO)",  codeobject, moddict, moddict); 
+             "(OOO)",  codeobject, moddict, moddict);
     return (res != 0) ? NULL : presult;     /* null if error in run_function */
 }
 // NOLINTEND
-
+// clang-format on

```

Comment:
Which criteria was used to prevent formatting of this file? (Just to know...)

Reply:
In general I don't want to format 3rd party code as this makes it very difficult to update to a newer version if needed.

Reply:
For this I thought that was the answer.  Thanks.

3rd party may even have a good but different formatting setting altogether and reformatting does not make sense.